### ISSUE TITLE
False Positive: aXe unable to determine color contrast within nested tags when font-weight is being applied in CSS.

### ISSUE THREAD

**Issue by GPFAFF** [dequelabs/axe-core#556]
Working with the aXe chrome browser extension I noticed some inconsistent behavior.  It is only present when we are applying `font-weight: bold` to a `<b>` or `<span>` inside any parent node.    When the node is on one line presentationally within the DOM, no color contrast flag, but when its container forces a line break it flags it as unable to determine color contrast. 

```
<p class="content">
   Lorem Ipsum <b>Bold Text</b> and a <b>lot more bold text</b>
</p>
```

Attached are pictures of the false positives. 

### offending markup

![screen shot 2017-10-02 at 3 23 54 pm](https://user-images.githubusercontent.com/1102168/31098087-1f279f5c-a787-11e7-86ae-7941966da571.png)

### associated false positive

![screen shot 2017-10-02 at 3 39 58 pm](https://user-images.githubusercontent.com/1102168/31098389-fb88c9f8-a787-11e7-92eb-7a9001fe25eb.png)

**Comment by marcysutton** [comment]
This is a known issue, but thanks for filing it! We have it filed internally but not on Github. And the details are very helpful!
